test/spinach/runner_test.rb

```diff
@@ -119,7 +119,50 @@
 
         runner.run.must_equal true
         feature.filename.must_equal filename
-        feature.line.must_equal line
+        feature.lines.must_equal [line]
+      end
+    end
+
+    describe "when lines set" do
+      let(:filename) { "features/cool_feature.feature" }
+      let(:line) { "12:24" }
+      let(:filenames) { ["#{filename}:#{line}"] }
+      let(:runner) { Spinach::Runner.new(filenames) }
+
+      before(:each) do
+        @feature_runner = stub
+        Spinach::Parser.stubs(:open_file).with(filename).returns parser = stub
+        parser.stubs(:parse).returns @feature = Spinach::Feature.new
+        Spinach::Runner::FeatureRunner.stubs(:new).
+          with(@feature, anything).
+          returns(@feature_runner)
+        runner.stubs(required_files: [])
+      end
+
+      it 'sets filename and lines on the feature' do
+        @feature_runner.stubs(:run).returns(true)
+        runner.run.must_equal true
+        @feature.filename.must_equal filename
+        @feature.lines.must_equal line.split(':').map(&:to_i)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.